composer.json

```diff
@@ -1,15 +1,17 @@
 {
-    "name": "dexpro-solutions-gmbh/softek-barcode-wrapper",
-    "description": "Wrapper arround the barcode SDK developed by Softek Software Ltd.",
-    "type": "library",
-    "license": "MIT",
-    "autoload": {
-        "psr-4": {
-            "DexproSolutionsGmbh\\SoftekBarcodeWrapper\\": "src/"
-        }
-    },
-    "require": {
-        "php": ">=7.4",
-        "ext-ffi": "*"
+  "name": "dexpro-solutions-gmbh/softek-barcode-wrapper",
+  "description": "Wrapper arround the barcode SDK developed by Softek Software Ltd.",
+  "type": "library",
+  "license": "MIT",
+  "autoload": {
+    "psr-4": {
+      "DexproSolutionsGmbh\\SoftekBarcodeWrapper\\": "src/"
     }
+  },
+  "require": {
+    "php": ">=7.4"
+  },
+  "suggest": {
+    "ext-ffi": "*"
+  }
```

Comment:
I think we should document in the README that FFI is a mandatory dependency for this package. It should also mention why we list FFI as suggested dependency.

Example:

```md

## Requirements

- `ext-ffi` - This package requires FFI. It is listed as suggested dependency .......

```

Reply:
done